src/Compilers/Core/Portable/DiagnosticAnalyzer/SuppressMessageAttributeState.TargetSymbolResolver.cs

```diff
@@ -55,26 +55,25 @@ private static string RemovePrefix(string id, string prefix)
                 return id;
             }
 
-            public void Resolve(IList<ISymbol> results)
+            public ImmutableArray<ISymbol> Resolve(out bool resolvedWithDocCommentIdFormat)
```

Comment:
@dotnet/roslyn-compiler Compiler changes are only in this file and the next file.

Reply:
can this be foc'ed?  i don't understand the out-param.

Reply:
Yes, added doc comment now.